doc/declarations.md

```diff
@@ -528,9 +569,53 @@ by rw [div]; rfl
 end Hide
 ```
 
+If Lean cannot find a permutation of the arguments for which all recursive calls are decreasing, it will print a table that contains, for every recursive call, which arguments Lean could prove to be decreasing. For example, a function with three recursive calls and four parameters might cause the following message to be printed
+
+```
+example.lean:37:0-43:31: error: Could not find a decreasing measure.
+The arguments relate at each recursive call as follows:
+(<, ≤, =: relation proved, ? all proofs failed, _: no proof attempted)
```

Comment:
Colons are inconsistent here: should there be one after the `?`?

The colons are't that great, in any case, as I'm tempted to misinterpret `=:` as an atom.

Reply:
Good point. How about putting the prose first and using parentheses?
```suggestion
Relation proved (<, ≤, =), all proofs failed (?), no proof attempted (_)
```